src/vdaf/hits.rs

```diff
@@ -202,6 +209,12 @@ impl<I> Hits<I> {
     }
 }
 
+impl<I> Clone for Hits<I> {
```

Comment:
Similar to the `Prio3` case, we can't derive `Clone` on `Hits` because of the `PhantomData<I>`.